src/components/NavigationBar/AppThemeToggle.tsx

```diff
@@ -1,6 +1,6 @@
 import { useAppTheme } from "../../context/AppThemeContext";
```

Comment:
add this in `navigationBar.tsx`:  `<li className='grid items-center'>
					<AppThemeToggle />
				</li>`   to center the icon